include/hgwosca.h

```diff
@@ -6,97 +6,88 @@ extern "C" {
 
 #include "stddef.h"
 
-double *const hgwosca(double(*obj)(const double *const, size_t),
-                      size_t wolf_count,
-                      size_t dim,
-                      size_t max_iterations,
-                      const double *const min_positions,
-                      const double *const max_positions);
+double * hgwosca(double(*obj)(const double *const, size_t),
```

Comment:
Do you think it would be a nice idea to somehow import the type definitions from `benchmark.c` into here so we can use the objective type and ensure consistency?

Reply:
Indeed, will look into it